src/apps/components/AppPermissionsDialog/AppPermissionsDialog.tsx

```diff
@@ -0,0 +1,149 @@
+import { AppPermissionsDialogConfirmation } from "@dashboard/apps/components/AppPermissionsDialog/AppPermissionsDialogConfirmation";
+import { AppPermissionsDialogPermissionPicker } from "@dashboard/apps/components/AppPermissionsDialog/AppPermissionsDialogPermissionPicker";
+import { useAppPermissionsDialogState } from "@dashboard/apps/components/AppPermissionsDialog/AppPermissionsDialogState";
+import { AppPermissionsDialogMessages } from "@dashboard/apps/components/AppPermissionsDialog/messages";
+import { useGetAvailableAppPermissions } from "@dashboard/apps/components/AppPermissionsDialog/useGetAvailableAppPermissions";
+import {
+  PermissionEnum,
+  useAppQuery,
+  useAppUpdatePermissionsMutation,
+} from "@dashboard/graphql";
+import useNotifier from "@dashboard/hooks/useNotifier";
+import { Dialog, DialogContent, DialogTitle } from "@material-ui/core";
+import { Skeleton } from "@material-ui/lab";
+import { Box, Text } from "@saleor/macaw-ui/next";
+import React, { useEffect } from "react";
+import { useIntl } from "react-intl";
+
+const messages = AppPermissionsDialogMessages.dialogRoot;
+
+interface OuterProps {
```

Comment:
```suggestion
interface AppPermissionsDialogProps {
```

Reply:
I don't export this interface, so IMO explicit namespace is redundant, WDYT?

Reply:
Yes it is but almost all Dashboard code follow this convention